amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -276,6 +323,56 @@ function compareAll(){
 										   value="&nbsp;&nbsp;<digi:trn>Compare All</digi:trn>&nbsp;&nbsp;"
 										   style="cursor: pointer; font-style: italic; float: right; margin: 0.5% 1.5% 0.5%;">
 								</c:if> <br>
+								
+				  
+				  <span style="cursor:pointer;font-style: italic;float:right;" onClick="toggleFilterSettings();" id="displayFilterButton">
+				  <digi:trn key="aim:Showfilteroptions">Show Filter options</digi:trn> &gt;&gt;</span>
+                                &nbsp;<br>
+								<div style="display:none;background-color:#ffffff;padding:2px" id="currentFilterSettings" >
+                                 <table cellpadding="2" cellspacing="2" border="0" width="250px">
+                                 <tr>
+                                 	<td align="right">
+                                 	<strong><digi:trn>Filter by:</digi:trn>&nbsp;&nbsp;</strong>
+                                	</td>
+                                	<td>
+                                 	<html:select property="filterBy" styleClass="inp-text" styleId="filterId" onchange="filterChanged(this.value);">
+                                 		<html:option value="User"><digi:trn>User</digi:trn> </html:option>
+                                 		<html:option value="Date"><digi:trn>Date</digi:trn> </html:option>
+                                 		<html:option value="Workspace"><digi:trn>Workspace</digi:trn> </html:option>
+                                 	</html:select>
+                                 	</td>
+                                 	<tr>
+                                 	<td align="right">
+                                 	<strong id ="filterBy" style="display:none"><digi:trn>Filter by:</digi:trn>&nbsp;&nbsp;</strong>
+                                	</td>
+                                 	<td>
+                                 	<html:select property="selectedUser" styleClass="inp-text" styleId="filteraction" style="display:none">

```

Comment:
in property for <html:select we hold the value for selectedUser. Should be a string, not a list. and is the property that we will test and from where we will take the value to filter

Reply:
you might also need to ad a Select option with -1 as value